doltpy/cli/write/write.py

```diff
@@ -88,27 +117,39 @@ def write_rows(dolt: Dolt,
     :param commit_date:
     :return:
     """
+
     def writer(filepath: str):
-        with open(filepath, 'w') as f:
-            fieldnames = set()
+        with open(filepath, "w") as f:
+            fieldnames: Set[str] = set()
```

Comment:
Is this type not inferred?

Reply:
I only changed lines that had syntax errors, so I'm assuming no? There are some weird rules around what mypy considers OK or not. The python author decides the source code, PEP suggestions and mypy rules so we sort of just have to play ball